The FD_ISSET macro and fd_set are defined in sys/select.h

--- a/src/prog/display-buttons.c	2012-10-26 23:21:38.000000000 +0200
+++ b/src/prog/display-buttons.c	2015-06-22 15:40:19.594034730 +0200
@@ -36,7 +36,8 @@
 #include <stdio.h>            /* printf()             */
 #include <time.h>             /* time()               */
 #include <errno.h>            /* errno                */
-#include <gpm.h>              /* gpm information      */
+#include <sys/select.h>       /* FD_ISSET             */
+#include "headers/gpm.h"      /* gpm information      */
 
 /* display resulting data */
 int display_data(Gpm_Event *event, void *data)
--- a/src/prog/display-coords.c	2012-10-26 23:21:38.000000000 +0200
+++ b/src/prog/display-coords.c	2015-06-22 15:41:05.433032252 +0200
@@ -37,7 +37,8 @@
 #include <stdio.h>            /* printf()             */
 #include <time.h>             /* time()               */
 #include <errno.h>            /* errno                */
-#include <gpm.h>              /* gpm information      */
+#include <sys/select.h>       /* FD_ISSET             */
+#include "headers/gpm.h"      /* gpm information      */
 
 /* display resulting data */
 int display_data(Gpm_Event *event, void *data)
--- a/src/prog/open_console.c	2012-10-26 23:21:38.000000000 +0200
+++ b/src/prog/open_console.c	2015-06-22 15:40:35.002033897 +0200
@@ -22,6 +22,7 @@
 #include "headers/message.h"        /* messaging in gpm */
 #include "headers/daemon.h"         /* daemon internals */
 #include <fcntl.h>
+#include <sys/select.h>
 
 int open_console(const int mode)
 {
